charmcraft/templates/init/requirements.txt.j2

```diff
@@ -1 +1 @@
-ops
+ops >= 1.2.0
```

Comment:
```suggestion
ops >= 1.2.0

```

Reply:
Agreed, I think that sounds like a nicer way to go for the future, but this should serve as a nice example in the mean time. Thanks :)